### PR TITLE
Assignment3

### PR DESCRIPTION
**Pull request by AlgorithmChopda**
Solved Assignment 3

1.  Get Index of Day
2.  Whats the Day
3. Word count
4. Return the slices

### CODE REVIEW

File: Assignment3/index_of_weekday.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	Input: 11
+	Output: Not a day
+*/
+
+package main
+
+import "fmt"
+
+func getDayFromIndex(index int) string {
+	var weekday_map = map[int]string{
```
Comment (Deepak1124) [inline Assignment3/index_of_weekday.go]: weekdayMap

File: Assignment3/whats_the_day.go [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+/*
+	Same question is asked as question 1
+*/
+
+package main
+
+import "fmt"
+
+func getDayFromIndex(index int) string {
+	var weekday_map = map[int]string{
```
Comment (Deepak1124) [inline Assignment3/whats_the_day.go]: weekdayMap

**Review by Deepak1124 — CHANGES_REQUESTED** [review]

File: Assignment3/index_of_weekday.go (hunk 1 above)
Comment (AlgorithmChopda) [reply Assignment3/index_of_weekday.go]: done

File: Assignment3/whats_the_day.go (hunk 2 above)
Comment (AlgorithmChopda) [reply Assignment3/whats_the_day.go]: done

**Pull request merged**